src/components/AuthenticatedLink.svelte

```diff
@@ -0,0 +1,18 @@
+<script>
+  import tippy from "./tippy";
+
+  export let href;
+</script>
+
+<style>
+  .lock-link {
+    text-decoration: none;
+  }
+</style>
+
+<a {href}><slot /></a>
+<a
+  class="lock-link"
+  href="https://docs.telemetry.mozilla.org/concepts/gaining_access.html"
+  use:tippy={{ content: "Access to this resource is limited to NDA'd Mozillians", placement: 'top' }}>
```

Comment:
```suggestion
  use:tippy={{ content: "Access to this resource is limited to NDA'd Mozillians", placement: 'right' }}>
```
Let's change `placement` to right so that the tooltip hover doesn't cover the bigquery link above it.

<img width="435" alt="CleanShot 2021-05-21 at 09 31 41@2x" src="https://user-images.githubusercontent.com/28797553/119170565-46847780-ba18-11eb-8e4d-3bbc581e0aa3.png">

Reply:
Hmm, I'm not sure -- this would be inconsistent with the behaviour of other on-hover tooltips (and tooltip behaviour in most apps) and I'm not sure if it's *too* much of a problem if the link is covered since you can just unhover the lock to see the link above. Do you feel strongly about this?

Reply:
I think a tooltip should not cover the object that it is attached to, which in this case it doesn't, so I don't feel strongly about this :) Tbh I never paid much attention to the tooltip placement so I wasn't aware that the default placement should be on top. Thanks for letting me know!